setup.py

```diff
@@ -55,6 +59,11 @@ def finalize_options(self):
 
 data = [path.join('capi', x) for x in libs if path.isfile(path.join('onnxruntime', 'capi', x))]
 
+#Adding python modules required for openvino ep
+python_modules_list = []
```

Comment:
is this needed?
shouldn't be a default in any case.

Reply:
This is not needed. Pushed patch to clean it.